src/django/api/helpers.py

```diff
@@ -47,12 +47,19 @@ def get_csv_values(csv_data):
     return values
 
 
+def try_parse_float(value):
```

Comment:
Consider naming this `try_parse_int_from_float` since that's what you're getting in the end.